### PR TITLE
Add more faqs in TiCDC

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? (Required)

- Add timezone issue in faq
- Add Ordering guarantees and transaction guarantees
- Fix some errors in command sample

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @leoppro @overvenus

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -56,3 +56,56 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --start-ts=415238226621235
...
+Warning: Unable to load '/usr/share/zoneinfo/zone.tab' as time zone. Skipping it.
+Warning: Unable to load '/usr/share/zoneinfo/zone1970.tab' as time zone. Skipping it.
+```
+
+如果是在特殊的公有云环境使用 MySQL，譬如阿里云 rds 并且没有修改 MySQL 的权限，就需要通过 `--tz` 参数指定时区。可以首先在 MySQL 查询其使用的时区，然后在创建同步任务和创建 cdc 服务时使用该时区。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+how variables like '%time_zone%';
```
Comment (overvenus) [inline ticdc/troubleshoot-ticdc.md:83]:
```suggestion
show variables like '%time_zone%';
```

File: ticdc/troubleshoot-ticdc.md [hunk 2]
```diff
@@ -56,3 +56,56 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --start-ts=415238226621235
...
 > TiCDC（4.0 发布版本）还不支持动态修改文件排序和内存排序。
+
+## 创建同步任务或同步到 MySQL 时遇到 `Error 1298: Unknown or incorrect time zone: 'UTC'` 错误
+
+这是因为下游 MySQL 没有加载时区，可以通过以下命令加载时区，加载后就可以正常创建任务或同步任务。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+mysql_tzinfo_to_sql /usr/share/zoneinfo | mysql -u root mysql -p
```
Comment (overvenus) [inline ticdc/troubleshoot-ticdc.md:67]: What is the "mysql_tzinfo_to_sql"?

Comment (amyangfei) [reply ticdc/troubleshoot-ticdc.md:67]: added a link to MySQL document

File: ticdc/ticdc-overview.md [hunk 3]
```diff
@@ -42,6 +42,29 @@ TiCDC 的系统架构如下图所示：
 - MySQL 协议兼容的数据库，提供最终一致性支持。
 - 以 TiCDC Open Protocol 输出到 Kafka，可实现行级别有序、最终一致性或严格事务一致性三种一致性保证。
 
+### 同步顺序保证和一致性保证
+
+#### 数据同步顺序
+
+- TiCDC 对于所有的 DDL/DML 都能对外输出**至少一次**
```
Comment (TomShawn) [inline ticdc/ticdc-overview.md:49]:
```suggestion
- TiCDC 对于所有的 DDL/DML 都能对外输出**至少一次**。
```

File: ticdc/troubleshoot-ticdc.md [hunk 4]
```diff
@@ -56,3 +56,56 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --start-ts=415238226621235
...
+
+```
+Enter password:
+Warning: Unable to load '/usr/share/zoneinfo/iso3166.tab' as time zone. Skipping it.
+Warning: Unable to load '/usr/share/zoneinfo/leap-seconds.list' as time zone. Skipping it.
+Warning: Unable to load '/usr/share/zoneinfo/zone.tab' as time zone. Skipping it.
+Warning: Unable to load '/usr/share/zoneinfo/zone1970.tab' as time zone. Skipping it.
+```
+
+如果是在特殊的公有云环境使用 MySQL，譬如阿里云 rds 并且没有修改 MySQL 的权限，就需要通过 `--tz` 参数指定时区。可以首先在 MySQL 查询其使用的时区，然后在创建同步任务和创建 cdc 服务时使用该时区。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:78]:
```suggestion
如果是在特殊的公有云环境使用 MySQL，譬如阿里云 RDS 并且没有修改 MySQL 的权限，就需要通过 `--tz` 参数指定时区。可以首先在 MySQL 查询其使用的时区，然后在创建同步任务和创建 TiCDC 服务时使用该时区。
```

**Comment by TomShawn** [comment]
LGTM

**Pull request merged**